tests/resources/defaultSettings.json

```diff
@@ -368,7 +368,7 @@
   },
   "CBC": {
     "Config": {
-      "CBCConfigEnable": false,
+      "CBCConfigEnable": true,
```

Comment:
I assume you modified these values so the RSA / CBC / NWI entries are added to the context menu, but you shouldn't modify these values (since this values should be false by default).  
What you should do instead is to leave these values as they are, and on the tests you add a `beforeAll` hook that will change these values on the local storage and then call the `ContextualMenu.update()` function.